2016-09-10-TWIT.md

```diff
@@ -20,14 +20,14 @@ Tessel Project leadership met up at Tesselcamp to determine the organization's g
 The two key goals for the next year are:
 
 1. **Grow the community to increase contributions and encourage both inclusion and accessibility to newcomers.**
-  * Build effective working groups that can complete their tasks with measurable results.
-  * Upgrade our documentation. Build more fritzing examples, API prototypes, and call out features that are currently missing from the docs.
-  * Update the tessel.io website to more accurately represent the project and its plans.
+   * Build effective working groups that can complete their tasks with measurable results.
```

Comment:
@frijol The file is apparently malformed Markdown (needs three spaces, not two...) I can remove this commit though